docs/quick_start/deploy_in_docker.md

```diff
@@ -1,44 +1,121 @@
-
-You need to download the docker compose yaml file of starrocks.
+## Step 1: Deploy
 
 ```sh
-wget https://raw.githubusercontent.com/StarRocks/starrocks/main/docker/docker-compose/docker-compose.yml
+docker run -p 9030:9030 -p 8030:8030 -p 8040:8040 -itd starrocks/allin1-ubuntu:2.5.4
 ```
 
-## Step 2: Run StarRocks cluster
+Then you can check the container status with:
+```sh
+docker ps
+```
 
-You should run the following command
+## Step 2: Connect to StarRocks
 
+StarRocks needs some time to get ready, it is recommended to wait at least 30 seconds before connecting.
 ```sh
-docker-compose up -d
+mysql -P9030 -h127.0.0.1 -uroot --prompt="StarRocks > "
 ```
 
-Then you can check the containers status.
+## Step 3: Using StarRocks
 
-## Step 3: Connect to StarRocks
+Use the following commands to check the status of FE and BE. If  `Alive` shows true for both FE and BE, StarRocks is healthy and ready to go.
 
-Connect to StarRocks cluster
+FE
+```SQL
+SHOW PROC '/frontends'\G
+```
+```plaintext
+StarRocks > SHOW PROC '/frontends'\G
+*************************** 1. row ***************************
+             Name: be552e5f0de9_9010_1680659932444
+               IP: be552e5f0de9
+      EditLogPort: 9010
+         HttpPort: 8030
+        QueryPort: 9030
+          RpcPort: 9020
+             Role: LEADER
+        ClusterId: 1556630880
+             Join: true
+            Alive: true
+ReplayedJournalId: 944
+    LastHeartbeat: 2023-04-05 02:49:36
+         IsHelper: true
+           ErrMsg: 
+        StartTime: 2023-04-05 01:59:00
+          Version: 3.0.0-RC01-a99fb8571
+1 row in set (0.05 sec)
 
 ```
-mysql -h 127.0.0.1 -P9030 -u root
+BE
+```SQL
+SHOW PROC '/backends'\G
+```
+```plaintext
+StarRocks > SHOW PROC '/backends'\G
+*************************** 1. row ***************************
+            BackendId: 10004
+                   IP: be552e5f0de9
+        HeartbeatPort: 9050
+               BePort: 9060
+             HttpPort: 8040
+             BrpcPort: 8060
+        LastStartTime: 2023-04-05 01:59:13
+        LastHeartbeat: 2023-04-05 02:50:06
+                Alive: true
+ SystemDecommissioned: false
+ClusterDecommissioned: false
+            TabletNum: 32
+     DataUsedCapacity: 4.474 KB
+        AvailCapacity: 197.683 GB
+        TotalCapacity: 235.983 GB
+              UsedPct: 16.23 %
+       MaxDiskUsedPct: 16.23 %
+               ErrMsg: 
+              Version: 3.0.0-RC01-a99fb8571
+               Status: {"lastSuccessReportTabletsTime":"2023-04-05 02:49:14"}
+    DataTotalCapacity: 197.683 GB
+          DataUsedPct: 0.00 %
+             CpuCores: 7
+    NumRunningQueries: 0
+           MemUsedPct: 0.24 %
+           CpuUsedPct: 0.5 %
+1 row in set (0.03 sec)
+
 ```
+Now we can create a table and insert some data.
 
-## Step4: Create table, Ingest data, Query
+**_NOTE:_** This quickstart starts with one BE, you need to add `properties ("replication_num" = "1")` in the CREATE TABLE clause so we only one replica of data is kept.
```

Comment:
so we only ...? sounds like missing some words in the `so ..` clause? or the `we` should be deleted?